jobs/buildhub/to_kinto.py

```diff
@@ -155,16 +102,53 @@ def fetch_existing(
             )
             previous_run_etag = '"%s"' % highest_timestamp
 
-    new_records = client.get_records(
-        _since=previous_run_etag,
-        pages=float('inf')
-    )
-
-    for record in new_records:
-        records[record['id']] = [
-            record['last_modified'],
-            hash_record(record)
-        ]
+    # The reason we can't use client.get_records() is because it is not
```

Comment:
If you want you can add a mention of https://github.com/Kinto/kinto-http.py/issues/175